docs/supported_docker_environment/continuous_integration/gitlab_ci.md

```diff
@@ -13,6 +13,8 @@ services:
 variables:
   # Instruct Testcontainers to use the daemon of DinD.
   DOCKER_HOST: "tcp://docker:2375"
+  # Instruct Docker not to start over TLS.
+  DOCKER_TLS_CERTDIR: ""
```

Comment:
Why not `DOCKER_TLS_CERTDIR: "/certs"` as specified in the docs?

Reply:
Original doc uses `DOCKER_HOST: "tcp://docker:2375"` which indicates TLS disabled mode.

I kept this sample as in TLS disabled mode.

See this docs section: https://docs.gitlab.com/ee/ci/docker/using_docker_build.html#tls-disabled

Reply:
@kiview any update please?